File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/http/HttpActivity.java
package com.jdroid.android.sample.ui.http;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class HttpActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return HttpFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/http/HttpActivity.kt
package com.jdroid.android.sample.ui.http

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class HttpActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return HttpFragment::class.java
    }
}